### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by SgtFoose** [firewill0/FIR_AWS_SystemDevelopment#47] [Ideas]
Dear Firewill,

First of all, I love your mods dude, I play every single day and always fly Firewill.
I have a few requests for future updates, hope you apricate them as much as I apricate your work!

1# By default all pods are GPS mode only. On large planes like the F-15, it's allot of work to put all pylons in DUAL mode.
It would be nice if we can send a preferred mode to multiple pylons at once. It would also be nice if the pylon modes could be integrated in the same page, maybe you could add a dropdown behind each pylon?

2# In game, while looking trough the LATIRN Pod (or others), I use Shift-t all the time to verify my targets with the in-game map (M key) to make sure my targets aren't friendly units. Some players are annoyed by this since they see my marker (shift-t). It would be great to have a similar feature that other players won't see.

Better: Build something to output if friendly units are near the marker and/or laser. This could be a line of text that says "nearest friendly 650m". It would also be great if we used the POI function in the I-TGT system to keep track of Friendly units near our POI's.
Let's say a Friendly units comes within 1km from one of our POI's, a line of text appears to tell us e.g. "Friendly units near POI 1".

3# This must be hard if not impossible, but would't it be great if I-TGT could show us a picture of the plane with all the pylons and their numbers? At least for all Firewill planes, this would be very helpful! The Pylon priority feature is great but almost unusable since it's very hard to know which pylon is witch bomb, especially when we install the same bomb on almost all pylons.

4# It would be very cool if ground units could mark targets and send them to the Firewill planes as suggested POI's /  Targets.
Most games have the laser designator to mark for CAS and Artillery, how cool would it be if Firewill would add a Firewill designator that can mark targets for planes?

I have a small bug report:

Every now and then when my plane crashes or when I die, the I-TGT POI's stay visible on the map. When I enter a new plane and make new POI's, they will overwrite but it's confusing to see old POI's that no longer exists.

---------------

Keep up the good work, send me your PayPal address so I can transfer you a small amount of money to thank you for your labour!

**Comment by firewill0** [comment]
Thank you for your kind word and suggestion!
Here's my answer for each question.
1. I'm consider to make some mode toggle button(GPS, Dual, CG and CG-T) for changing all pylons. I think it's will be good for plane with lot of pylons such as F-15E. 
2. someone already suggested similar feature, currently I'm consider to add some button for check friendly units activity around TGT marker.
3. I think It is considerable.
4. I have plan for make something similar like that already. you can see "MSG" in I-TGT UI, that is will doing something like what you say.
5. about marker bug, it's will be fixed in next AWS update.

right now, I'm busy with making new AMS UI and system, so hard to make promise when these feature comes out, but will do.

**Comment by wili114o** [comment]
In ARMA3 workshop steam, can you please add mq-1 UAV for this 

I you give free 3D modele

https://1drv.ms/u/s!AoKijKP8H6BNgX1ngdT-EbaUX2gF?e=LzIBZp

Thanks

**Reply by firewill0** [reply]
I'll consider of this, but right now pretty busy with others.

**Reply by wili114o** [reply]
Anyway, thanks for all the effort you put into this game, do you have a donation web page or a place to facilitate you with 3D model collection for you helping

I had to load the files in my cloud suddenly

**Reply by nourri47** [reply]
Hello I wanted to give you this tool

https://store.steampowered.com/app/777390/Flyout/

Do you want it
